crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -143,7 +143,7 @@ def transition(inst, name, state)
         else
           node_barclamp = node_role
         end
-        bc_databag = ProposalObject.find_data_bag_item("barclamps/#{node_barclamp}")
+        bc_databag = Chef::DataBag.load("barclamps/#{node_barclamp}") rescue nil
```

Comment:
@vuntz just to be clear: so everything inside 'installing' if block can go away?